examples/role.ts

```diff
@@ -0,0 +1,23 @@
+import { createClient } from '@clickhouse/client' // or '@clickhouse/client-web'
+
+void (async () => {
```

Comment:
Let's add a "See also" link to https://clickhouse.com/docs/en/interfaces/http#setting-role-with-query-parameters